torchtext/datasets/text_classification.py

```diff
@@ -44,19 +45,17 @@ def _csv_iterator(data_path, ngrams, yield_cls=False):
 def _create_data_from_iterator(vocab, iterator, include_unk):
```

Comment:
Same question as in [last PR](https://github.com/pytorch/text/pull/624#discussion_r350420442), is the name clear? Is this meant to follow the conventions established in the last PR?